Build/jenkins/Jenkinsfile_x86-64_windows

```diff
@@ -2,17 +2,18 @@
 pipeline {
     agent {label 'windows&&x64&&test'}
     environment {
-                JAVA_BIN='$WORKSPACE/openjdkbinary/j2sdk-image/jre/bin'
-                JCL_VERSION='current'
-                OPENJDK_TEST='$WORKSPACE/openjdk-test'
+    			JAVA_BIN='C:/Users/jenkins/workspace/openjdk_test_x86-64_windows/openjdkbinary/j2sdk-image/jre/bin'
```

Comment:
See issue #41 for jenkins there is an environment variable for this value WORKSPACE. However it doesn't work for jenkins pipeline environment section on windows. So this is temporary solution. Eventually we'd like using WORKSPACE variable.